src/append_only_data.rs

```diff
@@ -908,6 +1045,7 @@ impl Data {
         }
     }
 
+    /// Checks if the requester is the last owner.
```

Comment:
We need to explain why it is a Result and in which case an error might be returned.